koku/kafka_utils/test/test_utils.py

```diff
@@ -35,3 +35,37 @@ def test_kafka_connection_metrics_listen_for_messages(self, mock_start, mock_sle
         utils.is_kafka_connected(TEST_HOST, TEST_PORT)
         connection_errors_after = WORKER_REGISTRY.get_sample_value("kafka_connection_errors_total")
         self.assertEqual(connection_errors_after - connection_errors_before, 1)
+
+    def test_extract_from_header(self):
+        """Test test_extract_from_header."""
+        encoded_value = bytes("value", "utf-8")
```

Comment:
Another convention for indicating vars that hold bytes is a `b_` prefix.
```suggestion
        b_value = bytes("value", "utf-8")
```

Reply:
Bytes literals are UTF-8 if no encoding declaration exists in the file, so this could also be expressed as `b"value"`.

Reply:
I woulda committed this, but I didn't wanna wait on CI again.